scania-digital-design-system/sdds
# fix(banner): refactored events

**Describe pull-request**  
Refactored event names and removed aria props. Moved the aria props to be fetched on the component instead.

**Solving issue**  
Fixes: [AB#3367](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/3367)

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Banner -> Web Component
3. Check the readme

**Suggested test steps**

**Screenshots**  


**Additional context**
